test/assets/BaseConstants.scala

```diff
@@ -20,19 +20,18 @@ import v1.models.{CompanyNameModel, CountryCodeModel, LegalEntityIdentifierModel
 
 trait BaseConstants {
 
-  val ctutr                   = UTRModel("1123456789")
-  val sautr                   = UTRModel("1123456789")
-  val lei                     = LegalEntityIdentifierModel("ABCDEFGHIJKLMNOPQR12")
-  val invalidUtr              = UTRModel("1999999999")
-  val invalidShortUtr         = UTRModel("1")
-  val invalidLongUtr          = UTRModel("11234567890")
-  val companyName             = CompanyNameModel("Company Name ltd")
-  val companyNameMaxLength    = 160
-  val companyNameTooLong      = CompanyNameModel("a" * (companyNameMaxLength + 1))
-  val companyNameIsZero       = CompanyNameModel("")
-  val nonUkCountryCode        = CountryCodeModel("US")
-  val invalidCountryCode      = CountryCodeModel("AA")
-  val invalidEmptyCountryCode = CountryCodeModel("")
-  val invalidLei              = LegalEntityIdentifierModel("ABCDEFGHIJKLMNOPQRAA")
-
+  val ctutr: UTRModel                           = UTRModel("1123456789")
+  val sautr: UTRModel                           = UTRModel("1123456789")
```

Comment:
Yh saw this inconsistency and chose to ignore for now. Also, left it as is because it is that in the docs also.

<img width="643" alt="Screenshot 2023-02-07 at 10 19 04" src="https://user-images.githubusercontent.com/63714619/217217706-4536155c-a8d3-492a-ab85-1a7bbb0bbe5e.png">